torch/_inductor/compile_fx.py

```diff
@@ -325,8 +325,7 @@ def compile_fx_inner(
 
     # Inputs to fx_codegen_and_compile
     # Anything that affects codegen should go here, so if the signature
-    # of fx_codegen_and_compile changes, the list and dict should be updated accordingly
-    graph_args = [gm, example_inputs]
```

Comment:
See comment above. I changed this to be explicit about the two args (`gm`, `example_inputs`)